baybe/constraints/conditions.py

```diff
@@ -33,12 +36,31 @@ def _is_not_close(x: ArrayLike, y: ArrayLike, rtol: float, atol: float) -> np.nd
     return np.logical_not(np.isclose(x, y, rtol=rtol, atol=atol))
 
 
+def isclose(x: PolarsType, y: PolarsType, rtol: float, atol: float) -> Iterable[bool]:
```

Comment:
Output types should be as specific as possible, i.e. don't use `Iterable` but really say *which* kind of iterable.

Reply:
Fixed, changed to `np.ndarray`